src/containers/AccountModal/forms/Checkout.tsx

```diff
@@ -44,28 +48,61 @@ const Checkout = () => {
     return offerType === 'svod' ? addQueryParam(location, 'u', 'welcome') : removeQueryParam(location, 'u');
   }, [location, offerType]);
 
-  const couponCodeForm = useForm({ couponCode: '' }, async (values, { setSubmitting, setErrors }) => {
+  const paymentDataForm = useForm(
+    { couponCode: '', cardholderName: '', cardNumber: '', cardExpiry: '', cardCVC: '', cardExpMonth: '', cardExpYear: '' },
```

Comment:
It might make sense to keep coupon code form and payment submit form separate

Reply:
already done with the previous comment.